packages/wallet/Ledger.js

```diff
@@ -0,0 +1,100 @@
+const Eth = require('@ledgerhq/hw-app-eth');
+const Str = require('@ledgerhq/hw-app-str');
+const TransportNodeHid = require('@ledgerhq/hw-transport-node-hid');
+
+/**
+ * The class wraps api from @ledgerhq,
+ * with ES 6 async/aswait syntax
+ */
+class Ledger {
```

Comment:
Is it possible for `Account` to be powered by a Ledger/Trezor as native account?

One example use case:
- Developer creates new `Ledger` object and new `Account`, setAccount with Ledger, then proceed to call other `Account` methods as per normal.

The tricky thing is hardware wallet requires the manual physical approval step